core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneSet.java

```diff
@@ -78,16 +80,20 @@ public int addGeneSet(GeneSet geneSet) throws DaoException {
             
             con = JdbcUtil.getDbConnection(DaoGeneSet.class);
             pstmt = con.prepareStatement("INSERT INTO geneset " 
-                    + "(`ID`, `GENETIC_ENTITY_ID`, `EXTERNAL_ID`, `NAME_SHORT`, `NAME`, `REF_LINK`, `VERSION`) "
-                    + "VALUES(?,?,?,?,?,?,?)");
-            pstmt.setInt(1, geneSet.getId());
-            pstmt.setInt(2, geneSet.getGeneticEntityId());
-            pstmt.setString(3, geneSet.getExternalId());
-            pstmt.setString(4,geneSet.getNameShort());
-            pstmt.setString(5,geneSet.getName());
-            pstmt.setString(6, geneSet.getRefLink());
-            pstmt.setString(7, geneSet.getVersion());
+                    + "(`GENETIC_ENTITY_ID`, `EXTERNAL_ID`, `NAME_SHORT`, `NAME`, `REF_LINK`, `VERSION`) "
```

Comment:
@angelicaochoa here we removed ID field because I recently made it "auto increment"  in latest DB design. See https://github.com/thehyve/cbioportal/commit/7ca41555e3e2a8a7424432ade38dbf3aa9a72830. There are also a few other small DB changes in https://github.com/thehyve/cbioportal/commits/gsva_backend